scripts/advanced_quality_analysis.py

```
@@ -0,0 +1,270 @@
+        print("💡 Рекомендации:")
+        for rec in report["recommendations"]:
+            print(f"   {rec}")
+        print("="*60)
+    
+    def run_full_analysis(self):
+        """Запуск полного анализа качества кода"""
+        print("🚀 Запуск продвинутого анализа качества кода...")
+        print("="*60)
+        
+        # Запуск всех этапов анализа
+        tests_ok = self.run_tests()
```

Ruff (0.12.2)

Local variable `tests_ok` is assigned to but never used

Remove assignment to unused variable `tests_ok`

(F841)